modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -426,3 +532,31 @@ const NewOverflowList = () => {
   );
 };
 ```
+
+---
+
+### Menu (Main)
+
+We've updated the depth value of `Menu.Card` from `1` to `3`. This is a visually breaking change.
+
+---
+
+### Modal
+
+We've updated the `Modal` component to include new touch and responsive behaviors. The responsive
+`Modal` will be displayed when users are browsing on screen sizes between `320px` and `768px` and
```

Comment:
@mannycarrera4 Would love to get your thoughts on the Modal section. It wasn't included in the Upgrade Guide so I cobbled together this content based on the PR descriptions and code changes in #1700 and #1815. Let me know if I got anything wrong.